raco/myrial/parser.py

```diff
@@ -297,7 +297,8 @@ def add_python_udf(name, typ, **kwargs):
         :param typ: The output type of the function
         :type typ: string
         """
-        if name in Parser.udf_functions:
+        if name in Parser.udf_functions and \
+                not kwargs.pop('overwrite_if_exists', False):
```

Comment:
Yeah, I think it'd be clearer to reserve `kwargs` for the final `VariadicFunction`.